### ISSUE TITLE
Game option: make players be one-hit

### ISSUE THREAD

**Issue by yosoDev** [AllYourBlocks/ChunkWars#49]
Maybe with permanent strength-effect, will affect killing of mobs though

**Comment by AllYourBlocks** [comment]
Has been added to options panel, and has a scoreboard entry in the datapack - please explore how to do it @slipcor

**Comment by slipcor** [comment]
There is two ways to go about it. We can either raise players' attack strength, or we can lower players' health. Both is doable through the attribute command, and both will have effects on PvE combat.

We can test both options to see which one is better. One certainly is more challenging than the other :)

Here is the documentation of the attribute command, with more information about the possible values, maybe it sparks more ideas?

https://minecraft.fandom.com/wiki/Attribute

@AllYourBlocks

**Comment by AllYourBlocks** [comment]
I think I prefer raising players' strength, so that they can also one-hit mobs with a single punch if they can get close enough, but still be able to play 'normally' when interacting with terrain, taking fall damage, etc

Ideally we need the strength of players to be able to kill a player wearing fully enchanted Netherite Armour, with a single punch - so we should be looking to make the strength buff incredibly high.

Is the solution to use generic.attack_damage and set it to the maximum 2048 ?!

**Comment by slipcor** [comment]
Yes that is exactly what I planned to do. We will be able to test it in the next version. I was not planning on trying to find out a "safe" value, we will test the max value :D

**Comment by AllYourBlocks** [comment]
Relates to #117
